contracts/vaults/Deposit.sol

```diff
@@ -97,6 +96,7 @@ contract Deposit is Base {
         // if the user already has funds in the pool, give the previous reward
         if (_user.shares > 0) {
```

Comment:
we can get rid of passing `_user` , no?

Reply:
We can just load user again here but why though?

Reply:
let's keep the comments on topic @lirona  :)